streams/src/main/java/org/apache/kafka/streams/kstream/internals/foreignkeyjoin/SubscriptionWrapper.java

```diff
@@ -23,12 +23,15 @@
 
 
 public class SubscriptionWrapper<K> {
-    static final byte CURRENT_VERSION = 0;
+    static final byte CURRENT_VERSION = 1;
 
+    // v0
```

Comment:
nit:
```
// v0 fields:
```

Same for `v1` below

Reply:
https://github.com/apache/kafka/pull/11945/commits/f7f9181ec3312b21a3534919819ccd02dfdcbc74#diff-f9c7a23b1e3f38e1a7c00d05f8d5fa725df7a8147a0d44818d3b63cbf7b0ad23R28